systemd/amd64/jessie/entry.sh

```diff
@@ -28,7 +46,9 @@ if [ "$INITSYSTEM" = "on" ]; then
 	echo -e "${GREEN}Systemd init system enabled."
 	env > /etc/docker.env
 
-	echo -e "#!/bin/bash\n exec $@" > /etc/resinApp.sh
```

Comment:
@petrosagg, Updated!

Switching to this way because it behaves exactly as how docker handles CMD command. After doing some tests, the old solution behavior is not the same as docker when we use CMD in exec form to call a shell (ex `CMD [ "sh", "-c", "echo 123 345 456\'$%^£$%^" ]`)